README.md

```diff
@@ -48,18 +48,20 @@ It is possible to download files from AWS S3 as long as the proper AWS S3 url is
 
 | Key | Type | Required | Default | Description |
 | - | - | - | - | - |
+| baseFolder | string | Optional | /var/otp/graphs | The base directory for storing files. If `otpVersion` is set to 1.x, then an additional folder with the name set in `routerName` will be created and that additional folder will be where files are stored. If `otpVersion` is set to 2.x, then this folder will be used for storing files. |
```

Comment:
> The base directory for storing files. If `otpVersion` is set to 1.x, then a subdirectory named after `routerName` will be created and used for storing files e.g. `/var/otp/graphs/default`. If `otpVersion` is set to 2.x, then this folder itself will be used for storing files.

Reply:
Refactored in 2610ee9df6cb1878f067a5e2c1c277eed33836fe